main.js

```diff
@@ -492,7 +492,7 @@ class User {
     get fullName() {
         return `${this.firstName} ${this.lastName}`;
     };
-};
+}
 
 class Student extends User {
     constructor(admissionYear, courseName, firstName, lastName) {
```

Comment:
Try reversing the order of the arguments like this `constructor(firstName, lastName, admissionYear, courseName)`